src/v1/sources/shopify/webpixelTransformations/pixelTransform.js

```diff
@@ -116,15 +113,13 @@ function processPixelEvent(inputEvent) {
       break;
     case PIXEL_EVENT_TOPICS.CHECKOUT_STARTED:
     case PIXEL_EVENT_TOPICS.CHECKOUT_COMPLETED:
-      if (customer.id) message.userId = customer.id || '';
```

Comment:
We don't ever get this id in the pixel events in both anonymous or logged in users events. So it is better to remove it, and keep the userId assignment pointed to redis, thus ensuring a single source of truth.

Reply:
No, always seen null values populate there even if the user is logged in.